math/mathcore/inc/Math/Math_vectypes.hxx

```diff
@@ -0,0 +1,5 @@
+#include "RConfigure.h"
+#ifdef R__HAS_VECCORE
+#include <VecCore/VecCore>
+using Double_v = typename vecCore::backend::VcVector::Double_v;
```

Comment:
We should avoid putting Double_v in the global namespace.  What about putting in ROOT::Math  or at least ROOT::